coremltools/converters/mil/frontend/torch/ops.py

```diff
@@ -823,6 +823,58 @@ def adaptive_avg_pool2d(context, node):
 
     context.add(avg_pool)
 
+@register_torch_op
+def adaptive_max_pool2d(context, node):
+    inputs = _get_inputs(context, node, expected=2)
+
+    _input = inputs[0]
+    output_size = inputs[1].val
+    assert isinstance(output_size, _np.ndarray)
+    output_size = tuple(output_size)
+
+    if output_size == (1, 1):
+        # Represent (1,1) output size via @reduce_max
+        # Assume channel first ordering, reduce the last two (HW) dims.
+        axes = mb.const(val=[-2, -1], name=node.name + "_axes")
+        keep_dims = mb.const(val=True, name=node.name + "_keep_dims")
+
+        max_pool = mb.reduce_max(
+            x=_input, axes=axes, keep_dims=keep_dims, name=node.name
```

Comment:
`axes` and `keep_dims` can be fed directly instead of adding `const`
take a look at `stack` and `cat` op for reference

Reply:
You can directly do `axes=[-2, -1], keep_dims=True`

Reply:
I have made this change and pushed it to the my branch.